keps/sig-node/4540-strict-cpu-reservation/README.md

```diff
@@ -80,16 +83,37 @@ Admission is only comparing the cpu requests against the allocatable cpus. Since
 We propose to add a new `CPUManager` policy option called `strict-cpu-reservation` to the `static` policy of `CPUManager`.
 With this policy option, we remove the reserved cores from the list of all available cores at the stage of calculation DefaultCPUSet. As a result, burstable and best-effort containers are launched with a cpuset in which the reserved cores are excluded. 
 
+### User Stories (Optional)
+
+#### Story 1
+System daemons and interrupt processing are there to serve all processes including guaranteed workloads; they needs be isolated from bursty workloads.
+
+#### Story 2
+Isolating system daemons and interrupt processing is especially critical in all-in-one deployments where workloads are placed on combined master+worker(+HCI storage) nodes.
+
 ### Risks and Mitigations
 
 The risks of adding this new feature are quite low.
-It is isolated to a specific policy option within the `CPUManager`, and is protected both by the option itself, as well as the `CPUManagerPolicyAlphaOptions` feature gate (which is disabled by default).
+
+The feature is isolated to a specific policy option within the `CPUManager`, and is protected both by the option itself, as well as the `CPUManagerPolicyAlphaOptions` feature gate (which is disabled by default).
+
+When the feature is enabled, the size of the shared pool is reduced by that pod eviction may rise. This can be illustrated as follows:
+
+When `strict-cpu-reservation` is disabled:
+Total CPU cores (80) = Exclusively allocated (20) + Shared (60)
+
+When `strict-cpu-reservation` is disabled:
+Total CPU cores (80) = Exclusively allocated (20) + Reserved (8) + Shared (52)
+
+You want to make sure infrastructure pods are deployed with priorityClassName set to system-node-critical for their stability.
+
+Additionally, to avoid excessive eviction due to wrong dimensioning, this feature requires the number of reserved cores not exceeding half of the total cores.
```

Comment:
thing is, we're depending on proper user configuration with no safety net, and the feature offers little to none way to clearly identify the problem (e.g. "none of my best effort pods are running because guarenteed pods took all the CPUs"). I think we should explore more ways to prevent, or at least enable to report, this condition.

Reply:
Why do you think this is an issue?  "none of my best effort pods are running because guarenteed pods took all the CPUs" Can you elaborate your concern?